src/modules/tools/zprobe/ZProbe.cpp

```diff
@@ -123,9 +124,10 @@ void ZProbe::on_config_reload(void *argument)
     }
 
     this->probe_height  = THEKERNEL->config->value(zprobe_checksum, probe_height_checksum)->by_default(5.0F)->as_number();
-    this->slow_feedrate = THEKERNEL->config->value(zprobe_checksum, slow_feedrate_checksum)->by_default(5)->as_number(); // feedrate in mm/sec
-    this->fast_feedrate = THEKERNEL->config->value(zprobe_checksum, fast_feedrate_checksum)->by_default(100)->as_number(); // feedrate in mm/sec
+    this->slow_feedrate = THEKERNEL->config->value(zprobe_checksum, slow_feedrate_checksum)->by_default(1)->as_number(); // feedrate in mm/sec
```

Comment:
cannot change the default values here, they must be left as they were for backward compatibility. Breaking existing users use case is not nice.